inspect/README.md

```diff
@@ -0,0 +1,195 @@
+# forta-inspector-bot
+
+Inspects run environment capabilities and limits
+
+Moved from [forta-network/forta-inspector-bot](forta-inspector-bot-repo)
+
+# Inspection Details
+
+## Scan API Inspection
+
+### `scan-api.accessible`
+
+Checks Scan APIs accessibility by establishing rpc connection and asking for current block height
+
+`1` for accessible, `-1` for not
+
+### `scan-api.module.eth`
+
+The scan node should either declare `eth` module in `rpc_modules` method or it should return a valid response to
+a `eth_chainId` query
+
+`1` for supports, `-1` for not
+
+### `scan-api.module.net`
+
+The scan api should either declare `net` module in `rpc_modules` method or it should return a valid response to
+a `net_version` query
+
+`1` for supports, `-1` for not
+
+### `scan-api.block-by-number.hash`
+
+The hash of the block data retrieved from the scan API
+
+## Trace API Inspection
+
+### `trace-api.accessible`
+
+Checks trace APIs accessibility by establishing rpc connection and asking for current its chain id.
+
+`1` for accessible, `-1` for not
+
+### `trace-api.supported`
+
+Only necessary for Ethereum Mainnet.
+
+Returns `1` by default on other chains
+
+Checks trace APIs accessibility by establishing rpc connection and tracing the genesis block or the trace field in
+provided gRPC message
+
+`1` for accessible, `-1` for not
+
+### `trace-api.block-by-number.hash`
+
+The hash of the block data retrieved from the Trace API
+
+### `trace-api.trace-block.hash`
+
+The hash of the block trace data retrieved from the Trace API
+
+## Proxy Inspection
+
+### `proxy.accessible`
+
+Checks json-rpc proxy accessibility by establishing rpc connection and asking for current its chain id.
+
+`1` for accessible, `-1` for not
+
+### `proxy.chain-id`
+
+Chain ID of the provided json-rpc
+
+### `proxy.module.web3`
+
+The json-rpc proxy should either declare `net` module in `rpc_modules` method or it should return a valid response to
+a `net_version` query
+
+`1` for supports, `-1` for not
+
+### `proxy.module.eth`
+
+The json-rpc proxy should either declare `eth` module in `rpc_modules` method or it should return a valid response to
+a `eth_chainId` query
+
+`1` for supports, `-1` for not
+
+### `proxy.module.net`
+
+The json-rpc proxy should either declare `net` module in `rpc_modules` method or it should return a valid response to
+a `net_version` query
+
+`1` for supports, `-1` for not
+
+### `proxy.block-by-number.hash`
+
+The hash of the block data retrieved from the json-rpc proxy.
+
+### `proxy.history-support`
+
+Returns the earliest block height that proxy supports. Lower is better.
+
+## Network Inspection
+
+### `network.access.outbound`
+
+Tries to access https://api.forta.network
```

Comment:
Not doing this anymore - let's allow the consumer to have a broad understanding on this and let's remove the line.

Reply:
yeah looks kinda redundant since `speed.download` and `speed.upload` would be `-1` anyway. 
------
But, at some point, we could want to ditch speed tests and just go with outbound access check for a "lighter" inspection.  - since speed tests are the ones taking most of the inspection time

Reply:
Sorry never mind, we are actually doing it. I thought I removed it.